docs/DeveloperGuide.md

```diff
@@ -406,7 +407,7 @@ Use case ends.
 **Extensions**
 
 * 3a. The claim has already been closed for the client.
-    * 3a1. System shows an error message.
+    * 3a1. System shows that claim is closed with no error.
```

Comment:
The number system does not match here. Can change to the following:

``` markdown
* 1a. The claim has already been closed for the client.
    * 1a1. System shows that claim is closed with no error.
```

Reply:
Good catch!